R/unnest.R

```diff
@@ -52,6 +52,10 @@ unnest..tidytable <- function(.df,
                               names_sep = NULL,
                               names_repair = "unique") {
   vec_assert(.drop, logical(), 1)
+  
+  if (vec_is_empty(.df)) {
```

Comment:
You can remove this `vec_is_empty()` part - but can you edit `unnest_col()` so the beginning looks like this? This way it will return an empty logical column.

```r
.l <- pull.(.df, !!col)

.l <- list_drop_empty(.l)

if (length(.l) == 0) {
  .l <- list(logical())
}

.check_data <- .l[[1]]
is_vec <- is_simple_vector(.check_data)
```

The only new part is this:

```r
if (length(.l) == 0) {
  .l <- list(logical())
}
```

Reply:
I've made these changes in the latest commit. Does that look right?

PS Thanks for the wonderful package, keep up the good work! Hope dtplyr & tidyr catch wind of what you're doing here.

Reply:
Great, added the test case and updated the NEWS bullet. Appreciate the support.